.github/workflows/docker-build-and-push.yaml

```
@@ -62,6 +62,22 @@ jobs:
             *.cache-from=type=registry,ref=ghcr.io/${{ github.repository }}-buildcache:amd64-main
             *.cache-to=type=registry,ref=ghcr.io/${{ github.repository }}-buildcache:amd64-${{ github.ref_name }},mode=max
 
+      - name: Build 'autoware-tools'
+        uses: ./.github/actions/docker-build-and-push-tools
```

This job needs load-env and docker-build-and-push results.
```suggestion
      - name: Build 'autoware-tools'
        needs: [load-env, docker-build-and-push]
        uses: ./.github/actions/docker-build-and-push-tools
```